scripts/release_scripts/run_macaron.sh

```diff
@@ -559,8 +523,6 @@ proxy_var_names=(
     "HTTPS_PROXY"
     "FTP_PROXY"
     "NO_PROXY"
-    "MAVEN_OPTS"
```

Comment:
I think this section in the Sphinx documentation should be removed too after this change [Maven and Gradle proxy settings
](https://github.com/oracle/macaron/blob/8b8a29083b761689fdde8965a478d0b496f7a13e/docs/source/pages/installation.rst#L123-L141)